src/Rasterization/Renderers/EllipseRenderer.php

```diff
@@ -52,16 +52,16 @@ protected function renderStroke($image, $params, int $color, float $strokeWidth)
     {
         imagesetthickness($image, round($strokeWidth));
 
-        $width = $params['width'];
+        $width = (int)round($params['width']);
         if ($width % 2 === 0) {
             $width += 1;
         }
-        $height = $params['height'];
+        $height = (int)round($params['height']);
         if ($height % 2 === 0) {
             $height += 1;
         }
 
         // imageellipse ignores imagesetthickness; draw arc instead
-        imagearc($image, $params['cx'], $params['cy'], $width, $height, 0, 360, $color);
+        imagearc($image, (int)round($params['cx']), (int)round($params['cy']), (int)round($width), (int)round($height), 0, 360, $color);
```

Comment:
```suggestion
        imagearc($image, (int)round($params['cx']), (int)round($params['cy']), $width, $height, 0, 360, $color);
```

Probably no need to round/cast them twice 😄